pkg/cli/zip_cluster_wide.go

```diff
@@ -188,5 +188,10 @@ func (zc *debugZipContext) collectClusterData(
 		}
 	}
 
+	zc.clusterPrinter.info("collecting the inflight traces for jobs")
```

Comment:
I think we should wrap this in a default-true cli param (eg copy/pasta the --include-range-info flag handling)

Reply:
done